package.json

```diff
@@ -25,13 +25,13 @@
     "@angular/platform-browser-dynamic": "^5.2.0",
     "@angular/router": "^5.2.0",
     "core-js": "^2.4.1",
-    "ngx-twitter-timeline": "^0.1.4",
+    "ngx-twitter-timeline": "0.1.4",
     "normalize.css": "^8.0.0",
     "rxjs": "^5.5.6",
     "zone.js": "^0.8.19"
   },
   "devDependencies": {
-    "@angular/cli": "1.6.5",
+    "@angular/cli": "^1.6.5",
```

Comment:
Any reason?

Reply:
so, that if a higher version of angular 1.x.x can be used.